### DISCUSSION TITLE
not able to verify the contract on zkync

### DISCUSSION THREAD

**Discussion by SunflowerWeed** [Cyfrin/foundry-full-course-cu#2979] [Q&A]
Guys I am able to deploy the contract on zksync but I am not able to verify it. I need help please.

**Comment by EngrPips** [comment]
Hello @SunflowerWeed, it is not necessary for you to verify your contract on `zksync,` so I will say move and don't let it block you. But to be able to verify your contract, you must put your solidity file inside the `contracts` folder, and preferably you should be working in the `default-workspace`
